test/integration/full-content/fixtures/core__columns__deprecated.serialized.html

```diff
@@ -0,0 +1,23 @@
+<!-- wp:columns {"columns":3} -->
+<div class="wp-block-columns has-3-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:paragraph {"className":"layout-column-1"} -->
```

Comment:
The `className` being kept after the migration _may_ be unexpected, though conversely could better respect backward compatibility. I don't think it's a huge issue to keep, and also would be difficult to omit since the inner blocks are parsed before their parent, so a migration would otherwise need to re-parse and override.